source/install/install-kubernetes-mattermost.rst

```diff
@@ -61,7 +78,98 @@ Save the file as ``mattermost-installation.yaml``:
       username: ""
       password: ""
 
-**3. Edit the Installation Manifest File**
+Save the file as ``mattermost-installation.yaml``.
+
+**3. Apply the Installation Manifest File**
+
+To initiate deployment, apply the file, specifying the correct path, using:
+
+.. code-block:: sh
+
+  $ kubectl create ns mattermost
+  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+
+The deployment process can be monitored in the Kubernetes user interface.
+
+**4. Configure DNS and Use Mattermost**
+
+When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+
+.. code-block:: sh
+
+  $ kubectl -n mattermost get ingress
+
+Copy the resulting hostname or IP address from the ``ADDRESS`` column, open your browser, and connect to Mattermost.
+
+Use your domain registration service to create a canonical name or IP address record for the ``ingressName`` in your manifest,
+pointing to the address you just copied. For example, on AWS you would do this within a hosted zone in Route53.
+
+Navigate to the ``ingressName`` URL in your browser and use Mattermost.
+
+Restoring an Existing Mattermost MySQL Database
```

Comment:
I wonder whether this wouldn't be better off in the Install Cluster section of the guide because it covers installation/deployment and has a bit more complexity to it?

Reply:
Yes, I'd agree with that

Reply:
Changed it around to see whether it works better.